### PR TITLE
[Product-Partnerships] Aws lex documentation

### PR DESCRIPTION
**Pull request by aj-NR**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* This PR has the documentation for the newest integration with Amazon Lex
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by x8a — APPROVED** [review]
Thanks for adding this doc!

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
+
+## Features [#features]
+
+[Amazon Lex](https://aws.amazon.com/lex/) (Lex) is an AWS service for building conversational interfaces into applications using voice and text. With New Relic's Lex integration, you'll be able to monitor number of requests, channels used for Bot conversation and more. You'll be able to create custom charts of your data and [create alert conditions](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information) based on changes in Lex data or configuration.
+
+## Activate integration [#activate]
+
+To enable this integration follow standard procedures to [connect AWS services to New Relic](/docs/infrastructure/infrastructure-integrations/getting-started/connect-aws-integrations-infrastructure).
```
Comment (glennthomas1) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:25]: Change the link from the API Polling link you have to this one for CloudWatch Metric Streams: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/#set-up-metric-stream

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+## Configuration and polling [#polling]
```
Comment (glennthomas1) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:27]: Remove this section as it is not relevant

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+These are the default [polling](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) values for the Amazon EBS integration:
+
+* New Relic polling interval: 15 minutes
+* Amazon CloudWatch data interval: [1 minute or 5 minutes](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ebs-metricscollected.html), depending on volume type
+
+## Find and use data [#find-data]
+
+To find this integration's data, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS** and select one of the Amazon Lex integration links.
+
+You can [query and explore your data](/docs/using-new-relic/data/understand-data/query-new-relic-data) using the `LexSample` [event type](/docs/data-apis/understand-data/new-relic-data-types/#event-data), with a `provider` value of `Lex`.
```
Comment (glennthomas1) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:40]: Is there really a LexSample event table? I thought everything ends up in Metric now?

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx (hunk 1 above)
Comment (aj-NR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:25]: done

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx (hunk 3 above)
Comment (aj-NR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:40]: All other documentation is also maintained like this. This LexSample is defined in infra-metrics file

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx (hunk 2 above)
Comment (aj-NR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:27]: done

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx (hunk 3 above)
Comment (aj-NR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx:40]: addressed

**Pull request merged**